spec/factories/user_locations.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user_location do
+    user_id {1}
+    location_id {1}
```

Comment:
Try this: https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations